sdcflows/workflows/base.py

```diff
@@ -105,22 +105,29 @@ def init_sdc_estimate_wf(fmaps, epi_meta, omp_nthreads=1, debug=False):
         return workflow
 
     workflow.__postdesc__ = """\
-Based on the estimated susceptibility distortion, a corrected
-EPI (echo-planar imaging) reference was calculated for a more
-accurate co-registration with the anatomical reference.
+Based on the estimated susceptibility distortion, a corrected EPI (echo-planar imaging) reference
+was calculated for a more accurate co-registration with the anatomical reference.
 """
 
     only_syn = 'syn' in fmaps and len(fmaps) == 1
 
-    # PEPOLAR path
+    # PEPOLAR path (now topup)
     if 'epi' in fmaps:
-        from .pepolar import init_pepolar_unwarp_wf, check_pes
+        from .topup import init_topup_wf, check_pes
```

Comment:
I don't love the `topup` name of the module. I would prefer this workflow to be within the `pepolar` module.